docs/neural_modeling/plot_01_current_injection.py

```diff
@@ -284,10 +284,11 @@
 # Now, let's convert the binned spikes into the firing rate, by smoothing them
 # with a gaussian kernel. Pynapple again provides a convenience function for
 # this:
-
+#
```

Comment:
This changes how the following lines are rendered, from comment to markdown. I think they belong as comments, because they're just explaining the specific details of the arguments (so they're not conceptually important). What do you think?